src/main/resources/db/migration/V1_0__ADD_STAFF_TABLES.sql

```diff
@@ -0,0 +1,31 @@
+-- Install the pg_trgm extension
+CREATE EXTENSION IF NOT EXISTS pg_trgm;
+
+
+drop table if exists staff;
+
+CREATE TABLE staff(
+    id SERIAL PRIMARY KEY,
+    first_name VARCHAR NOT NULL,
+    last_name VARCHAR NOT NULL,
+    job_title VARCHAR,
+    email VARCHAR NOT NULL
+);
+
+-- CONCURRENTLY builds the index without locking out writes
+CREATE INDEX CONCURRENTLY staff_email_idx ON staff USING gin (email gin_trgm_ops);
+
+drop table if exists staff_temp;
+
+CREATE TABLE staff_temp(
```

Comment:
Just a note that I think you can use the `LIKE` keyword, i.e. `CREATE TABLE TableB (LIKE TableA INCLUDING ALL); `

..it's not particularly useful here but I think it will be useful when doing the swap